Makefile

```diff
@@ -69,8 +69,9 @@ dockerhub-login:
 	docker login -u $(DOCKER_USERNAME) -p $(DOCKER_PASSWORD)
 
 version-set:
```

Comment:
i am thinking if we can remove this `version-set` command.. can we adjust version on the flight ? without making this command required to be executed

Reply:
I think we should save this option. It's convenient for changing tags in docker-compose files for testing environments. It isn't  necessary step during deploing